metadata-ws/src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/entities/Taxonomy.java

```diff
@@ -17,8 +17,10 @@
  */
 package uk.ac.ebi.ampt2d.metadata.persistence.entities;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import io.swagger.annotations.ApiModelProperty;
+import org.hibernate.annotations.Formula;
```

Comment:
can be removed.

Reply:
You're right, removed this and other unused annotations in https://github.com/EBIvariation/metadata-ws/pull/53/commits/f50f715fb67cb7453c0ca920d5ddfecf3a3a37da.